src/app/node-data/component.ts

```diff
@@ -176,8 +176,10 @@ export class NodeDataComponent extends BaseFormValidator implements OnInit, OnDe
       .subscribe(_ => (this._nodeDataService.operatingSystemSpec = this._getOperatingSystemSpec()));
 
     this._settingsService.adminSettings.pipe(takeUntil(this._unsubscribe)).subscribe(settings => {
```

Comment:
If we would change this pipe to `take(1)` then the bug should be gone. The service is responding very quickly as admin settings are cached but the problem is that there is refresh happening sometimes.

Reply:
The initialization should happen only once anyways so there is no point of keeping this subscription open.

Reply:
ah you're right. Always forget `take(1)` :see_no_evil:
Changed 👍